kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/AbstractServicesUi.java

```diff
@@ -607,7 +608,8 @@ public void onChange(ChangeEvent event) {
                     }
                 } catch (NumberFormatException e) {
                     this.valid.put(param.getId(), false);
-                    result.add(new BasicEditorError(box, box.getText(), e.getLocalizedMessage()));
+                    result.add(new BasicEditorError(box, box.getText(),
```

Comment:
the try catches only issues for numeric fields https://github.com/eclipse/kura/pull/1189/files#diff-e55d221b480c9c188b8d72740e7d2e46R533
I don't see a problem if we say that "foo" is invalid because out of range. But we can simply write that Value "foo" is invalid. I don't think that is worth implementing so much logic for such an improper value.